test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGet.CommandLine.Test.csproj

```diff
@@ -36,10 +36,9 @@
       <Project>{32a23995-14c7-483b-98c3-0ae4185373ea}</Project>
       <Name>NuGet.Credentials</Name>
     </ProjectReference>
-
-    <ProjectReference Include="..\TestExtensions\TestableCredentialProvider\TestableCredentialProvider.csproj" ReferenceOutputAssembly="false" />
-    <ProjectReference Include="..\..\..\test\TestExtensions\SampleCommandLineExtensions\SampleCommandLineExtensions.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="$(TestUtilitiesDirectory)Test.Utility\Test.Utility.csproj" />
+    <ProjectReference Include="..\..\TestExtensions\SampleCommandLineExtensions\SampleCommandLineExtensions.csproj" />
+    <ProjectReference Include="..\..\TestExtensions\TestablePluginCredentialProvider\TestableCredentialProvider.csproj" />
```

Comment:
Add ``ReferenceOutputAssembly="false"`` to both of these. They are only consumed by the xcopy below, not at compile time